webpack/components/extensions/HostDetails/Tabs/ErrataTab.js

```diff
@@ -29,9 +31,13 @@ import './ErrataTab.scss';
 
 export const ErrataTab = () => {
   const hostDetails = useSelector(state => selectAPIResponse(state, 'HOST_DETAILS'));
-  const { id: hostId } = hostDetails;
+  const {
+    id: hostId,
+    content_facet_attributes: contentFacetAttributes,
+  } = hostDetails;
+  const contentFacet = propsToCamelCase(contentFacetAttributes);
   const dispatch = useDispatch();
-
+  const [errataInstallable, setErrataInstallable] = useState(true);
```

Comment:
I'm a fan of more explicit values here
```suggestion
  const toggleGroupStates = ['all', 'installable'];
  const [all, installable] = toggleGroupStates;
  const [toggleGroupState, setToggleGroupState] = useState(installable);
```